hs_core/tasks.py

```diff
@@ -134,21 +134,24 @@ class HydroshareTask(Task):
     retry_backoff = True
     retry_backoff_max = 600
     retry_jitter = True
-    soft_time_limit = 7200  # 2 hours - allows for graceful shutdown
-    time_limit = 7260  # 2 hours + 1 minute - hard limit
+    soft_time_limit = 30  # 30 sec for testing
+    time_limit = 45  # 45 sec for testing
 
-    # Set the time limit to be the greater of 2hrs or the specific task limits
-    time_limit = max(time_limit,
-                     settings.NIGHTLY_RESOURCE_REPAIR_DURATION,
-                     settings.NIGHTLY_GENERATE_FILESYSTEM_METADATA_DURATION
-                     )
+    # # Set the time limit to be the greater of 2hrs or the specific task limits
+    # time_limit = max(time_limit,
+    #                  settings.NIGHTLY_RESOURCE_REPAIR_DURATION,
+    #                  settings.NIGHTLY_GENERATE_FILESYSTEM_METADATA_DURATION
+    #                  )
 
 
 @celery_app.on_after_finalize.connect
 def setup_periodic_tasks(sender, **kwargs):
     if (hasattr(settings, 'DISABLE_PERIODIC_TASKS') and settings.DISABLE_PERIODIC_TASKS):
         logger.debug("Periodic tasks are disabled in SETTINGS")
     else:
+        # every min for testing
+        sender.add_periodic_task(crontab(minute='*'), nightly_repair_resource_files.s(),
+                                 options={'queue': 'periodic'})
```

Comment:
@sblack-usu in testing this PR locally, attempted to kick off async tasks every minute.
When I look at Flower, I don't see that any tasks get started.
When I look at redPanda console, I can see that the `periodic` topic has been created, but there aren't any messages in the topic.

When I manually kick off an async task:
`nightly_repair_resource_files.apply_async(())`
It gets created in its own topic in RPC, that's fine. It still doesn't show up in FLOWER fyi so we might just want to get rid of flower...?

But I'm bringing this to your attention because it really seems to me that the periodic tasks might not be running which is relevant as we test [3.11.0](https://github.com/hydroshare/hydroshare/issues/6100)

Reply:
hmm when I check 3.11.0 staged on beta, I can see the periodic tasks in the Flower queue.
So maybe this is just an issue locally...